src/android/CameraPreviewFragment.java

```diff
@@ -286,8 +286,10 @@ public void torchSwitch(boolean torchOn, TorchCallback torchCallback) {
             camera.getCameraControl().enableTorch(torchOn).get();
             torchCallback.onEnabled(null); 
         } catch (Exception e) {
-            torchCallback.onEnabled(new Exception("Failed to switch " + (torchOn ? "on" : "off") + " torch", e));
+            torchCallback.onEnabled(new Exception("Failed to switch " + (torchOn ? "on" : "off") + " torch: " + e.getMessage()));
```

Comment:
```suggestion
            torchCallback.onEnabled(new Exception("Failed to switch " + (torchOn ? "on" : "off") + " torch: " + e.getMessage(), e));
```

This should be also be passed right?